judge/models/profile.py

```diff
@@ -68,7 +64,7 @@ def get_users_url(self):
         return reverse('organization_users', args=(self.id, self.slug))
 
     class Meta:
-        ordering = ['key']
+        ordering = ['slug']
```

Comment:
Can we just use the name to order? Otherwise people are going to be like "why isn't the organizations sorted properly????"